tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -14,10 +14,9 @@ advanced/D#
                  kind: TYPE
                  properties: 4
                  display_name: "_"
-     self {
```

Comment:
I'm not sure what's happening there 🤔

Reply:
This isn't a diff introduced by the PR though.